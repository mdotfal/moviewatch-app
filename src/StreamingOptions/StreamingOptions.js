import React, { Component } from 'react';
import './StreamingOptions.css';

class StreamingOptions extends Component {
  render() {
    return (
      <div className="content-streaming overview-section flex">
        <fieldset>
          <legend>Select all that apply</legend>
          <div>
            <label htmlFor="is_netflix">Netflix
            <input type="checkbox" id="is_netflix" name="option" value="Netflix" /></label>
          </div>
          <div>
            <label htmlFor="is_hulu">Hulu
            <input type="checkbox" id="is_hulu" name="option" value="Hulu" /></label>
          </div>
          <div>
            <label htmlFor="is_prime">Prime Video
            <input type="checkbox" id="is_prime" name="option" value="Prime Video" /></label>
          </div>
        </fieldset>
      </div>
    );
  };
};

export default StreamingOptions;