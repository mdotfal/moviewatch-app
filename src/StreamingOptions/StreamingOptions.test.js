import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import StreamingOptions from './StreamingOptions';

describe( `StreamingOptions component`, () => {
  it(Â `renders without errors`, () => {
    const wrapper = shallow( <StreamingOptions /> )
    expect( toJson( wrapper )).toMatchSnapshot()
  })
})