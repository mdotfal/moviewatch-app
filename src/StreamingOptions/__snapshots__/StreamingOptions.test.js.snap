// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StreamingOptions component renders without errors 1`] = `
<div
  className="content-streaming overview-section flex"
>
  <fieldset>
    <legend>
      Select all that apply
    </legend>
    <div>
      <label
        htmlFor="is_netflix"
      >
        Netflix
        <input
          id="is_netflix"
          name="option"
          type="checkbox"
          value="Netflix"
        />
      </label>
    </div>
    <div>
      <label
        htmlFor="is_hulu"
      >
        Hulu
        <input
          id="is_hulu"
          name="option"
          type="checkbox"
          value="Hulu"
        />
      </label>
    </div>
    <div>
      <label
        htmlFor="is_prime"
      >
        Prime Video
        <input
          id="is_prime"
          name="option"
          type="checkbox"
          value="Prime Video"
        />
      </label>
    </div>
  </fieldset>
</div>
`;
